azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -55,8 +56,8 @@
   private final boolean creatorDefaultPermissions;
   private final ConcurrentHashMap<Integer, Project> projectsById =
       new ConcurrentHashMap<>();
-  private final ConcurrentHashMap<String, Project> projectsByName =
-      new ConcurrentHashMap<>();
+  private final CaseInsensitiveConcurrentHashMap<Project> projectsByName =
+      new CaseInsensitiveConcurrentHashMap<>();
```

Comment:
Is it worth adding a new unit test to make sure that the cache stays consistent with the DB schemas?

Reply:
Added the test for project cache.

Reply:
Thanks